docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -256,6 +259,37 @@ difference is that you would specify your Hibernate ORM configuration in `META-I
 </persistence>
 --
 
+== Defining entities in external projects or jars
+
+Hibernate ORM in Quarkus relies on compile-time bytecode enhancements to your entities. If you define your entities in the
```

Comment:
I am not sure if this is 100% correct. 

If you have an additional module in the same project that does not have beans.xml then I don't know if this will be handled correctly currently. Also adding beans.xml will have the same effect in terms of turning the jar into an application archive. 

Thinking about the current state of affairs there is likely a mismatch between what gets indexed in dev mode and producing the actual artifact.

Reply:
So, do I need to file an issue about this or what?

Reply:
OK, you were right, the bean file is enough. Updated the docs.